crates/bevy_audio/src/audio.rs

```diff
@@ -115,10 +118,137 @@ where
             settings,
             sink_handle: id,
             source_handle: audio_source,
+            spatial: None,
         };
         self.queue.write().push_back(config);
         Handle::<AudioSink>::weak(id)
     }
+
+    /// Play audio from a [`Handle`] to the audio source, placing the listener at the given
+    /// transform, an ear on each side separated by `gap`. The audio emitter will placed at
+    /// `emitter`.
+    ///
+    /// `bevy_audio` is not using HRTF for spatial audio, but is transforming the sound to a mono
```

Comment:
`bevy_audio` is not using HRTF for spatial audio, but is transforming the sound to a mono track, and then changing the level of each stereo channel according to the distance between the emitter and each ear by amplifying the difference between what the two ears hear.